CONTRIBUTING.md

```diff
@@ -39,7 +39,7 @@ make test
 Then before submitting your PR make sure the code quality follows the standards. You can run the following command to format:
 
 ```bash
-make commit
+make precommit
 ```
 
```

Comment:
For some reason I can't suggest the closing of the code block, but you get the idea?
```suggestion

Make sure to install `pre-commit` before running the command:
```bash
pip install pre-commit
```